src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -399,6 +399,20 @@ public TagletWriterImpl getTagletWriterInstance(TagletWriterImpl.Context context
         return new TagletWriterImpl(this, context);
     }
 
+    /**
+     * {@return true if the page written by this writer should be indexed,
```

Comment:
Use `{@code true}`

Reply:
Thanks, but this is a backport, which I would like to keep this in sync with mainline: https://github.com/openjdk/jdk/blob/cf948548c390c42ca63525d41a9d63ff31349c3a/src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java#L415

I wonder if we want to change this in mainline and backport separately?